scripts/api-compatibility-check/README.md

```diff
@@ -0,0 +1,83 @@
+# API Compatibility check
+
+## What is it
+
+API compatibility checks help us to ensure that we would not accidentally break a public API. The set of scripts runs Apple's tool `swift-api-digester` to dump public API to a reusable JSON file and compare each other.
+
+## How to run
+
+You can find two scripts in this folder. The first, `breaking-api-check.sh`, designed to run set of public API checks over the pre-built baseline. The second, `rebuild-baseline.sh`, helps to generate new baseline based on git reference. Let's take a closer look on each of them.
+
+### `breaking-api-check.sh`
+
+The `breaking-api-check.sh` accept optional argument `-p path` which should be packaged version of MapboxMaps. Without `-p` argument, script would trigger `scripts/release/packager/package-mapbox-maps.sh` to build XCFramework from scratch. It is useful to path pre-built binary to avoid extra building steps. Breaking API checks running against `.baseline.zip` JSON API digester dump. To update the baseline, you have to proceed to `rebuild-baseline.sh`.
+The script is path-independent so you can call it from any other folder with an absolute or relative path.
```

Comment:
```suggestion
`breaking-api-check.sh` accepts an optional argument `-p path` which should be a packaged version of MapboxMaps. Without the `-p` argument, the script triggers `scripts/release/packager/package-mapbox-maps.sh` to build XCFramework from scratch. It is useful to specify the path of a pre-built binary to avoid the extra building steps. Breaking API checks running against the `.baseline.zip` JSON API digester dump. To update the baseline, see `rebuild-baseline.sh`.

The script is path-independent so you can call it from any other folder with an absolute or relative path.
```